ios/SnapCloneXcode/test_runner.sh

```
@@ -0,0 +1,34 @@
+#\!/bin/bash
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)